refactoringminer/src/main/java/gr/uom/java/xmi/decomposition/AbstractCall.java

```diff
@@ -159,7 +159,15 @@ private boolean identicalExpressionAfterTypeReplacements(AbstractCall call, Set<
             String expression2 = call.getExpression();
             String expression1AfterReplacements = expression1;
             for (Replacement replacement : replacements) {
-                if (replacement.getType().equals(ReplacementType.TYPE)) {
+                if (replacement.getType().equals(ReplacementType.TYPE) ||
+                    //allow only class names corresponding to static calls
+                    (replacement.getType().equals(ReplacementType.VARIABLE_NAME) &&
+                        Character.isUpperCase(expression1.charAt(0)) && Character.isUpperCase(expression2.charAt(0)))) {
+
+                    if (replacement.getBefore().equals(expression1) && replacement.getAfter().equals(expression2)) {
+                        return true;
+                    }
+
                     expression1AfterReplacements = ReplacementUtil.performReplacement(expression1AfterReplacements, expression2, replacement.getBefore(), replacement.getAfter());
```

Comment:
line is longer than 120 symbols

Reply:
There are many such lines in the code that has not been rewritten. I'm not refactoring such issues. Because it takes a long time to fix them all. And may be performed when somebody plan to change something in that code